docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-designing-a-case-proc.adoc

```diff
@@ -1,2 +1,101 @@
++
+* `name` - the name of the case. This is usually the subject of the case or project that is being case managed.
+* `package` - the location in which case file is to be created.
+* `Case ID prefix (optional)` - a configurable prefix that allows you to easily distinguish different types of cases. The prefix is follows with a generated ID in the format `ID-XXXXXXXXXX`, where `X` is a generated number that provides a unique ID for the case instance. If a prefix is not provided, the default prefix is `CASE` and generates instances with the following identifiers: 
++
+`CASE-0000000001`
++
+`CASE-0000000002`
++
+`CASE-0000000003`
++
+and so on. If a prefix is set to something else, such as `IT` for example, the following identifiers are generated:
++
+`IT-0000000001`
++
+`IT-0000000002`
++
+`IT-0000000003`
+
++
+. Click *Ok* to open the case designer.
++
+The designer creates a `WorkDefinition` file that includes the work items required for case definition. These appear as usable objects in the case designer *Object Library*, accessible by clicking the image:3897.png[left arrows] in the upper-left corner of the designer.
++ 
+** Verify that the case designer includes the `Milestones` and `Cases` objects by either checking the *Object Library* dock on the left side of the case designer, or by checking the `WorkDefinition` file under *Work Item Definitions* in the *Project Explorer* window on the left side of the {CENTRAL} workbench.
+. *Define process level and case level variables.* 
++
+Click the image:3140.png[] arrow on the right side of the designer to open the `Properties (BPMN-Diagram)` panel. Click the down arrow next to `Variable Definitions` to open the `Editor for Variable Definitions`.
++
+.IT Orders variables
+image::case-variables.png[Editor for IT Orders variables]
++
+In the IT Orders sample project, you can see that certain variables are identified as `Case File`. A case file variable is stored in the case file itself, and promotes reuse of the case file data rather than copying it across multiple process instances. For example, the `hwSpec` variable is available to all processes within the case, and is accessible directly from the case file even without an existing process instance. 
++
+For more information about the case file, see <<case-management-case-file-con-{context}>>.
+
+. *Define the roles involved in the case.*
+.. Click image:3898.png[] to open the *Properties* menu on the right side of the designer and open the *Editor for Case Roles*. 
+.. Click *Add Case Role* to add case roles.  
++
+The IT Orders sample case management project includes the following roles:
++
+
+.ITOrders Case Roles
+image::case_roles.png[Case Roles]
+
++
+* `owner` - the employee who is making the hardware order request. There can be only one of these roles.
+* `manager` - the employee's manager; the person who will approve or deny the requested hardware. There is only one manager in the IT Orders project.
```

Comment:
this is a bit confusing as there are not managers defined in the project at all. It's only case role with cardinality set - that means when case instance is started it can have only one person/group assigned to given role.

Reply:
Changed to:
`manager` - the employee's manager; the person who will approve or deny the requested hardware. The role cardinality is set to `1`, which means that only one person or group can be assigned to this role.